### ISSUE TITLE
Fix Hangfire stopping error

### ISSUE THREAD

**Issue by mscottford** [corgibytes/freshli-cli#362]
The following error shows up in the output for most commands:
```
2022/10/21 14:07:34.730| WARN|Hangfire.Server.BackgroundProcessingServer:30|Stopping the server due to DomainUnload or ProcessExit event...
2022/10/21 14:07:34.740| WARN|Hangfire.Server.BackgroundServerProcess:30|Server 658ccef5-70dc-4cce-94b9-63fe6bafaf26 caught shutdown signal...
2022/10/21 14:07:34.740| WARN|Hangfire.Server.BackgroundServerProcess:30|Server 658ccef5-70dc-4cce-94b9-63fe6bafaf26 stopped non-gracefully due to CountersAggregator, Worker, DelayedJobScheduler, RecurringJobScheduler. Outstanding work on those dispatchers could be aborted, and there can be delays in background processing. This server instance will be incorrectly shown as active for a while. To avoid non-graceful shutdowns, investigate what prevents from stopping gracefully and add CancellationToken support for those methods.
```

I think this is because the Hangfire server isn't getting shut down properly.

**Comment by mscottford** [comment]
This isn't a critical error, because I'm pretty sure there are no jobs in the queue when this is happening.

**Comment by mscottford** [comment]
I suspect this can be solved by adding a `System.CommandLine` middleware or perhaps adjusting the middleware that is setting the number of workers. If we move the code to start the Hangfire server to the middleware section that runs before the command, we can stop the server after the command runs.

**Comment by donabp** [comment]
Hi @mscottford , I've been working on trying to get this bug resolved however I am a bit stuck. I understand the last comment but could you clarify what you mean by implementing `System.CommandLine` middleware? I've been looking through google where other forums have said that they had to "hack" this together to get it running properly:
https://discuss.hangfire.io/t/ability-to-stop-all-the-server-instances-during-deployments/2285/4
https://stackoverflow.com/questions/66582783/how-to-restart-hangfire-server-or-stop-it-and-start-a-new-one-to-change-the-qu

**Comment by donabp** [comment]
I'm still testing the issue out on how to get rid of this warning. I've tried implementing all the options from the above forums as well:


![Image](https://user-images.githubusercontent.com/50669675/199015190-4f94a044-1b10-4567-ab3d-6064380b0150.png)

**Comment by mscottford** [comment]
This was fixed by #407.
